packages/panna-sdk/src/core/util/api-service.ts

```diff
@@ -227,6 +235,79 @@ export class PannaApiService {
       throw error;
     }
   }
+
+  /**
+   * Retrieves the status of an onramp.money session
+   *
+   * This function polls the onramp.money API to get the current status of a fiat-to-crypto
+   * onramp session. The status includes transaction details, amounts, and any error messages.
+   *
+   * @param params - Parameters for retrieving the session status
+   * @param params.sessionId - The onramp.money session identifier
+   * @param params.authToken - Optional JWT token for authentication
```

Comment:
The `authToken` is mandatory.
```suggestion
   * @param params.authToken - JWT token for authentication
```